sunbird-rc-esignet-integration-impl/pom.xml

```diff
@@ -92,6 +92,20 @@
 			<version>1.7.30</version>
 			<scope>provided</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.apache.velocity/velocity -->
+		<dependency>
+			<groupId>org.apache.velocity</groupId>
+			<artifactId>velocity</artifactId>
+			<version>1.7</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20210307</version>
```

Comment:
this version has vulnerabilities, https://mvnrepository.com/artifact/org.json/json, use the latest one if required

Reply:
updated to lastest version as of now